.buildkite/pipeline.e2e.yml

```diff
@@ -0,0 +1,19 @@
+agents:
+  queue: agent-runners-linux-amd64
+
+env:
+  CI_E2E_TESTS_TARGET_ORG: agent-e2e-testing
+  CI_E2E_TESTS_TARGET_CLUSTER: e1773a06-b43c-4b77-84e5-5e088a737f6a
```

Comment:
Would it be good to log a task in linear to remove these two?

Reply:
Oh ok. Sure.